src/components/NavBar.jsx

```diff
@@ -48,7 +51,7 @@ const NavBar = () => {
         <button className="h-8 flex justify-center items-center pt-1">
           <img
             className="w-8"
-            src={menuIcon}
+            src={navbarTest}
```

Comment:
Is this import supposed to be named test or was it a temp thing? If temp, rename to a more descriptive name